src/OpenTelemetry.Instrumentation.Dependencies/Implementation/SqlClientDiagnosticListener.cs

```diff
@@ -22,6 +22,9 @@ namespace OpenTelemetry.Instrumentation.Dependencies.Implementation
 {
     internal class SqlClientDiagnosticListener : ListenerHandler
     {
+        internal const string ActivitySourceName = "System.Data.SqlClient";
```

Comment:
This can get callbacks from both System.Data.SqlClient and Microsoft.Data.SqlClient (yes both uses same diagnosticsource name :unamused:).
Do we need to use separate ActivitySource? I don't see a strong need, but letting you know of this, if you weren't aware.

Reply:
I did consider this, but couldn't think of a reason why we would need multiple ActivitySources. Right now, .NET Core System.Data.SqlClient (DiagnosticSource) and/or Microsoft.Data.SqlClient (DiagnosticSource), and .NET Framework System.Data.SqlClient (EventSource) will all publish their Activity objects through a single ActivitySource. To make it more clear what's going on, I just added "OpenTelemetry." namespace in front of our .NET Framework HttpWebRequest, StackExchange.Redis, & SQL instrumentation (now OpenTelemetry.SqlClient) ActivitySource names. LMK what you think about that.